# HG changeset patch
# User Michael Goffioul <michael.goffioul@gmail.com>
# Date 1344158109 -3600
#      Sun Aug 05 10:15:09 2012 +0100
# Node ID a182972304fa6227256babb1bd5bb2c61e78c6ca
# Parent  a1bcffac7fa8576dc4e2a2e37b79952c991ab70c
Include config.h when it exists and is specified.

diff --git a/libqterminal/unix/kpty.cpp b/libqterminal/unix/kpty.cpp
--- a/libqterminal/unix/kpty.cpp
+++ b/libqterminal/unix/kpty.cpp
@@ -17,16 +17,20 @@
    Library General Public License for more details.
 
    You should have received a copy of the GNU Library General Public License
    along with this library; see the file COPYING.LIB.  If not, write to
    the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
    Boston, MA 02110-1301, USA.
 */
 
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+
 #include "unix/kpty_p.h"
 
 #ifdef __sgi
 #define __svr4__
 #endif
 
 #ifdef __osf__
 #define _OSF_SOURCE
